packages/alba/alba.v0.4.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "alba"
+version: "v0.4.0"
+maintainer: "Helmut Brandl <helmut.brandl@gmx.net>"
+authors: "Helmut Brandl <helmut.brandl@gmx.net>"
+homepage: "http://albatross-lang.sourceforge.net"
+license: "GPLv2"
+dev-repo: "git+https://github.com/hbr/albatross.git"
+bug-reports: "http://github.com/hbr/albatross"
+build: [
+  [make "build"]
+]
+depends: [
+  "ocamlfind"
+  "ocamlbuild"
```

Comment:
The two above dependencies should probably be flagged as ```{build}``` dependencies. This avoids rebuilding everything in case they are upgraded and this should be fine as I suppose that you are using those two tools as tools and not as libraries.